Week1/Exersise2-W1/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace Exersise2_W1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int i;
+            int j;
+            for (i = 1; i <= 10; i++)
+            {
+                for (j = 1; j <=9; j++) 
+                {
+                    int ji = j * i;
+                    Console.Write($"|{j} x {i} = {ji}\t");
```

Comment:
@Tri Don't make it complicated. we just need syntax `{j * j,3}` which inserts a 3-character string. In this case, the string representation of the first object is right-aligned in the 3-character field.